src/components/images/imageLoader.js

```diff
@@ -1,50 +1,83 @@
-define(['lazyLoader', 'imageFetcher', 'layoutManager', 'browser', 'appSettings', 'userSettings', 'require', 'css!./style'], function (lazyLoader, imageFetcher, layoutManager, browser, appSettings, userSettings, require) {
-    'use strict';
+import * as lazyLoader from 'lazyLoader';
+import * as userSettings from 'userSettings';
+import 'css!./style';
+/* eslint-disable indent */
 
-    var self = {};
+    export function lazyImage(elem, source = elem.getAttribute('data-src')) {
+        if (!source) {
+            return;
+        }
 
-    function fillImage(elem, source, enableEffects) {
+        fillImageElement(elem, source);
+    }
 
-        if (!elem) {
-            throw new Error('elem cannot be null');
+    export function fillImage(entry) {
+        if (!entry) {
+            throw new Error('entry cannot be null');
         }
 
-        if (!source) {
-            source = elem.getAttribute('data-src');
+        var source = undefined;
+        if (entry.target) {
+            source = entry.target.getAttribute('data-src');
+        } else {
+            source = entry;
```

Comment:
I think that if we get here, we will fail in `fillImageElement(entry.target, source);`